tests/VObject/EmptyParameterTest.php

```diff
@@ -51,6 +53,7 @@ public function testVCard21Parameter(): void
         $vcard = new Component\VCard([], false);
         $vcard->VERSION = '2.1';
         $vcard->PHOTO = 'random_stuff';
+        /* @phpstan-ignore-next-line 'Cannot call method add() on string' */
         $vcard->PHOTO->add(null, 'BASE64');
```

Comment:
how can we call `->add` on a property, which we assigned a `string` shortly before?

is this related to magic `__get` & `__set` or similar, and we should/can use a extension in phpstan for that?
https://phpstan.org/developing-extensions/class-reflection-extensions

Reply:
yes, there is "magic property"  stuff happening, so PHOTO does not really end up being a string!
I haven't dug deep into this. I was not being too concerned about the test code.
But I will probably have to dig into it when I get to the "real" code.